skyve-ee/src/skyve/modules/admin/ControlPanel/views/external/edit.xml

```diff
@@ -46,8 +46,8 @@
 	</tabPane>
 	
 	<actions>
-		<download className="DownloadClientSAIL" displayName="Download" inActionPanel="false" toolTip="Generate SAIL jar for SAIL client" />
-		<action className="GenerateTestData" displayName="Generate Test Data" inActionPanel="false" iconStyleClass="fa fa-plus" clientValidation="false"/>
-		<action className="DeleteTestData" displayName="Delete Test Data" inActionPanel="false" iconStyleClass="fa fa-trash" clientValidation="false"/>
+		<download className="DownloadClientSAIL" displayName="admin.controlPanel.actions.downloadClientSail.actionName" inActionPanel="false" toolTip="admin.controlPanel.action.downlaodClientSail.toolTip" />
```

Comment:
Hey @brandon-klar , the tooltip key is spelt incorrectly, can you fix that up? I don't think it will work currently, looks correct in the i18n.properties file